shell/edit/kubevirt.io.virtualmachine/VirtualMachineVolume/type/__tests__/HarvesterEditExisting.test.ts

```diff
@@ -0,0 +1,102 @@
+import { mount } from '@vue/test-utils';
+import HarvesterEditExisting from '@shell/edit/kubevirt.io.virtualmachine/VirtualMachineVolume/type/existing.vue';
+import { _EDIT } from '@shell/config/query-params';
+
+describe('component: HarvesterEditExisting', () => {
+  it('should display all the inputs', () => {
+    const wrapper = mount(HarvesterEditExisting, {
+      propsData: {
+        mode: _EDIT, value: {}, rows: []
+      },
+      mocks:      {
+        $store: {
+          getters: {
+            'harvester/all': () => [],
+            'i18n/t':        jest.fn()
+          }
+        }
+      }
+    });
+
+    const inputWraps = wrapper.findAll('[data-testid^=input-hee-]');
+
+    expect(inputWraps).toHaveLength(5);
+  });
+
+  it.each([
+    'name',
+  ])('should emit an update on %p input', (field) => {
+    const wrapper = mount(HarvesterEditExisting, {
+      propsData: {
+        mode: _EDIT, value: {}, rows: []
+      },
+      mocks:      {
+        $store: {
+          getters: {
+            'harvester/all': () => [],
+            'i18n/t':        jest.fn()
+          }
+        }
+      }
+    });
+    const input = wrapper.find(`[data-testid="input-hee-${ field }"]`).find('input');
+    const newValue = 123;
+
+    input.setValue(newValue);
+
+    expect(wrapper.emitted('update')).toHaveLength(1);
+  });
+
+  // TODO: Restore this test when the input is allowed to edit
```

Comment:
Is this TODO still valid? Does it need an associated issue?

Reply:
Yes, I should actually mention it: https://github.com/rancher/dashboard/issues/6179